tests/failover.go

```diff
@@ -165,7 +165,8 @@ func (oa *OperatorActions) TruncateSSTFileThenCheckFailover(info *TidbClusterCon
 
 	// checkout pd config
 	var pdCfg *pdapi.PDConfigFromAPI
-	if tc.IsHeterogeneous() {
+	// same code in two branch on purpose..?
```

Comment:
I think L170 should be updated.

Reply:
https://github.com/pingcap/tidb-operator/pull/3947/commits/0544d2b7fa27a9c8257e46008fd9608717fdfacb

Reply:
Remove this line?